dev.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
```

Comment:
```suggestion
#!/usr/bin/env sh
```
Is from what I have heard the more accepted way of writing this type of shebang nowadays